Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -202,6 +203,37 @@ public class Snapshotter {
 
         return compositeImage
     }
+
+    // MARK: - Camera
+
+    /// Returns the coordinate bounds corresponding to a given `CameraOptions`
+    ///
+    /// - Parameter camera: The camera for which the coordinate bounds will be returned.
+    /// - Returns: `CoordinateBounds` for the given `CameraOptions`
+    public func coordinateBounds(for camera: CameraOptions) -> CoordinateBounds {
+        return mapSnapshotter.coordinateBoundsForCamera(
+            forCamera: MapboxCoreMaps.CameraOptions(camera))
+    }
+
+    /// Calculates a `CameraOptions` to fit a list of coordinates.
+    ///
+    /// - Parameters:
+    ///   - coordinates: Array of coordinates that should fit within the new viewport.
+    ///   - padding: The new padding to be used by the camera.
+    ///   - bearing: The new bearing to be used by the camera.
+    ///   - pitch: The new pitch to be used by the camera.
+    /// - Returns: A `CameraOptions` that fits the provided constraints
+    public func camera(for coordinates: [CLLocationCoordinate2D],
+                       padding: UIEdgeInsets,
+                       bearing: Double?,
+                       pitch: Double?) -> CameraOptions {
+        return CameraOptions(
+            mapSnapshotter.cameraForCoordinates(
```

Comment:
same as above, this formatting looks weird. this should go up a line and everything else should be indented accordingly